External/Plugins/CodeRefactor/Commands/RenameFile.cs

```diff
@@ -48,17 +48,19 @@ protected override void ExecutionImplementation()
                     return;
                 }
             }
+
+            newPath = Path.Combine(Path.GetDirectoryName(oldPath), newPath);
+
             // refactor failed or was refused
             if (Path.GetFileName(oldPath).Equals(newPath, StringComparison.OrdinalIgnoreCase))
             {
                 // name casing changed
                 string tmpPath = oldPath + "$renaming$";
                 File.Move(oldPath, tmpPath);
-                oldPath = tmpPath;
+                File.Move(tmpPath, newPath);
+                DocumentManager.MoveDocuments(oldPath, newPath);
             }
-            if (!Path.IsPathRooted(newPath)) newPath = Path.Combine(Path.GetDirectoryName(oldPath), newPath);
```

Comment:
It's moved to [line 52](https://github.com/fdorg/flashdevelop/pull/1291/files/fa9a81c728ffdc8cac598b5fbb5a9997cd111fd3#diff-b28e14b0a502fd942600df3ec02eb1ebR52), without the `if` statement, since `Path.Combine()` already checks if `path2` is rooted.